src/main/java/org/apache/mesos/hdfs/state/LiveState.java

```diff
@@ -71,25 +73,23 @@ public void updateTaskForStatus(Protos.TaskStatus status) {
       // If initializing the first NN or reconciling the first NN or bootstrapping the first NN
       // set the status to initialized
       if (status.getMessage().equals(HDFSConstants.NAME_NODE_INIT_MESSAGE)
```

Comment:
Can't we just remove the "useless parens"? Or if you do have to do NOPMD, is there a way to do NOPMD(useless_parens) to specify that only that rule should be ignored on this line, rather than all rules?